fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/WitnessMessage.java

```diff
@@ -0,0 +1,76 @@
+package com.github.dedis.student20_pop.model;
+
+import java.util.Collections;
+import java.util.Set;
+
+/**
+ * Class to model a message that needs to be signed by witnesses
+ */
+public class WitnessMessage {
+
+    private String messageId;
+    /**
+     * Base 64 URL encoded ID of the message that we want to sign
+     */
+    private Set<String> witnesses;
+    /**
+     * Set of witnesses that have signed the message
+     */
+    private String title = "";
+    /**
+     * Title that will be displayed for the message
+     */
+    private String description = ""; /** Description that will be displayed for the message*/
+
+    /**
+     * Constructor for a  Witness Message
+     *
+     * @param messageId ID of the message to sign
+     */
+    public WitnessMessage(String messageId) {
+        witnesses = Collections.emptySet();
+        this.messageId = messageId;
+    }
+
+    /**
+     * Method that checks whether the current message has been signed by the Witness public key provided
+     *
+     * @param pk public key of the witness
+     */
```

Comment:
Nice doc but where's the method? 😅

Reply:
Yeah in the end this method was deleted just forgot to delete the doc .